server/odc-service/src/main/java/com/oceanbase/odc/service/connection/CloudMetadataClient.java

```diff
@@ -70,4 +85,9 @@ public interface CloudMetadataClient {
     OBTenantEndpoint getTenantEndpoint(@NotBlank String instanceId, @NotBlank String tenantId);
 
     OBDatabaseUser getSysTenantUser(@NotBlank String instanceId);
+
+    enum PermissionAction {
```

Comment:
RamPermissionAction?

Reply:
I assume that it is not related to RAM in the interface layer. RAM is just an authentication-check implementation.

Reply:
CloudPermissionAction, cause that PermissionAction is too common to misunderstand it.

Reply:
> CloudPermissionAction? cause that PermissionAction is too common to misunderstand it.

okay, done~